go/tasks/plugins/k8s/spark/spark.go

```diff
@@ -199,6 +199,10 @@ func (sparkResourceHandler) BuildResource(ctx context.Context, taskCtx pluginsCo
 		j.Spec.MainClass = &sparkJob.MainClass
 	}
 
+	if taskCtx.TaskExecutionMetadata().IsInterruptible() {
```

Comment:
:( you have to update this everytime, I hope SparkOperator takes a pod spec like thing?

Reply:
Yeah, Spark 3.0 does support pod template but needs it to be in the image. Need to explore that path a bit more.